_data/simple-icons.json

```
@@ -17872,7 +17872,7 @@
         },
         {
             "title": "Wii U",
-            "hex": "8B8B8B",
+            "hex": "0097C9",
```

```suggestion
            "hex": "0096C8",
```

Color pickers may have differences according to your monitor display profile. Please use the `#0096C8` from the SVG source code.